### PR TITLE
[16.0] [OU-ADD] attachment_indexation: Nothing to do

### PR DESCRIPTION
**Pull request by NgoHoangHung**

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -110,7 +110,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | crm_livechat                                    | Nothing to do        |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| crm_mail_plugin                                 |                      |                                                 |
+| crm_mail_plugin                                 | Nothing to do        |                                                 |
```
Comment (hoangtiendung070797) [inline docsource/modules150-160.rst]: @NgoHoangHung Change is happening in PR https://github.com/OCA/OpenUpgrade/pull/4063

**Comment by pedrobaeza** [comment]
It seems your latest rebases over the PRs are not correct, because you have an outdated remote.

**Comment by NgoHoangHung** [comment]
I will fix it soon

**Pull request closed without merging**